src/controllers/attachment/index.ts

```diff
@@ -84,12 +96,11 @@ export class attachment extends Controller {
   public async get(): Promise<Attachment[]> {
     this.log.debug('retrieving all attachment')
 
-    const attachments: Attachment[] = await this.db.attachment()
+    const attachments: DbAttachment[] = await this.db.attachment()
     return attachments.map(
-      ({ binary_blob, created_at, ...rest }: any): Attachment => ({
+      ({ created_at, ipfs_hash, ...rest }): Attachment => ({
```

Comment:
add an eslint comment for the deliberate unused variable?

Reply:
This is such a common pattern I'm going to suggest updating our eslint rules